### DISCUSSION TITLE
Will support for writing m29f160 based gbc flash carts be added in the future?

### DISCUSSION THREAD

**Discussion by WilliamDingler** [sanni/cartreader#441] [Ideas & Suggestions]
I just purchased an m29f160-based gbc flash cart from insidegadgets and was wondering if support for this IC would be added in a later version. Are there any plans to add additional IC support to the cart reader?

**Comment by Icesythe7** [comment]
Just ask @lesserkuma to add support for it, he manages the software for FlashGBX and I built him a sanni cart so I'm sure he would add support for it.
